src/components/infoBlock/InfoBlock.tsx

```diff
@@ -93,6 +93,42 @@ function InfoBlockList({ items, inline }: InfoBlockContentListProps) {
   );
 }
 
+type InfoBlockCollapseProps = {
+  icon: React.ReactElement;
+  items: React.ReactElement[];
+  className?: string;
+  title: string;
+  titleClassName?: string;
+};
+
+function InfoBlockCollapse({
```

Comment:
HDS now has the Accordion component, and with it they introduced the `useAccordion` hook for custom accordion components. Maybe we should use that so that we have better guarantees that the accordion is accessible?

https://hds.hel.fi/components/accordion#custom-accordion

Reply:
That makes sense. I totally forgot that HDS had that component.